pkg/core/native/native_test/management_test.go

```diff
@@ -566,13 +566,15 @@ func TestManagement_GetContract(t *testing.T) {
 	t.Run("by ID, positive", func(t *testing.T) {
 		managementInvoker.Invoke(t, si, "getContractById", cs1.ID)
 	})
+	/* C# compatibility
```

Comment:
Are we planning to create an issue in C# repo? If not, then maybe we don't need this test at all?

Reply:
Yes, there will be a patch there.